code/controllers/CronTaskController.php

```diff
@@ -79,16 +79,36 @@ public function isTaskDue(CronTask $task, \Cron\CronExpression $cron) {
 	 * @param SS_HTTPRequest $request
 	 */
 	public function index(SS_HTTPRequest $request) {
-		// Check each task
+		
+		$params = $this->request->allParams();
+
+		$action = $params['Action'];
```

Comment:
I'd suggest changing the routes to make it explicit what is being executed. Maybe like this?`only/$CronTaskClass`.